package.json

```diff
@@ -19,7 +19,8 @@
     "prettier": "prettier --write './**/*.js'",
     "pretty-check": "prettier --check './**/*.js'",
     "test": "truffle test",
-    "build": "yarn compile && yarn networks-inject",
+    "build": "yarn compile && yarn copy-artifacts && yarn networks-inject",
+    "copy-artifacts": "cp ./node_modules/@gnosis.pm/dex-contracts/build/contracts/BatchExchange.json ./build/contracts",
```

Comment:
Why is this needed? So that we don't need to run a migration step and still can access the artefact with the direct require?

Reply:
Yes! People just wanting to setup up bracket strategies should not go through the migration step